### PR TITLE
i18n(fr): Add `integrations-guide/preact` in French language

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
i18n(fr): Add `integrations-guide/preact` in French language

#### Related issues & labels (optional)

- Linked to https://github.com/withastro/docs/issues/6052
- Suggested label: i18n

### CODE REVIEW

**Comment by thomasbnt** [comment]
But what happened to my branches to make me partytown here...

**Review by dreyfus92 — COMMENTED** [review]
Bon boulot @thomasbnt, juste une petite suggestion 😁

**Comment by thomasbnt** [comment]
Merci @dreyfus92 !

File: src/content/docs/fr/guides/integrations-guide/preact.mdx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+Cette **[intégration Astro][astro-integration]** permet le rendu côté serveur et l'hydratation côté client pour vos composants [Preact](https://preactjs.com/).
+
+## Pourquoi Preact ?
+
+Preact est une bibliothèque qui vous permet de créer des composants d'interface utilisateur interactifs pour le web. Si vous souhaitez créer des fonctionnalités interactives sur votre site à l'aide de JavaScript, vous préférerez peut-être utiliser son format de composant plutôt que d'utiliser directement les API du navigateur.
+
+Preact est également un excellent choix si vous avez déjà utilisé React. Preact fournit la même API que React, mais dans un format beaucoup plus petit de 3 Ko. Il supporte même le rendu de nombreux composants React en utilisant l'option de configuration `compat` (voir ci-dessous).
+
+**Vous souhaitez en savoir plus sur Preact avant d'utiliser cette intégration ?**\
```
Comment (Yan-Thomas) [inline src/content/docs/fr/guides/integrations-guide/preact.mdx:20]:
```suggestion
**Vous souhaitez en savoir plus sur Preact avant d'utiliser cette intégration ?**
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🙌

**Pull request merged**